src/services/review/ReviewRequestService.ts

```diff
@@ -108,6 +111,11 @@ export default class ReviewRequestService {
           )
           // NOTE: We ignore the errors and use a placeholder
           .unwrapOr("")
+        const fileUrl = await pageNameRes
+          .andThen((pageName) =>
+            this.pageService.retrieveCmsPermalink(pageName, siteName)
+          )
+          .unwrapOr("")
```

Comment:
Hmm I am confused here. Could you explain why we return an empty string here if it fails, rather than throwing/returning an error?

Reply:
going to raise a PR later on to resolve this and make the type parsing clearer

Reply:
#662 - parsing logic clarified so that if we fail any preconditions to parse a file, we return a minimal object back to the frontend. 

this is because we can fail to parse any one file but still able to parse others successfully; this is abit of a cop-out because we choose to mix a file that failed to parse w/ successful parsing files but the FE is still able to display (some) relevant info